source/features/unreleased-commits.tsx

```diff
@@ -96,7 +101,7 @@ async function init(signal: AbortSignal): Promise<false | void> {
 
 	await api.expectToken();
 
-	observe(branchSelectorParent, add, {signal});
+	observe(branchSelector, add, {signal});
```

Comment:
We generally try to keep compatibility with both versions wherever possible, or else we get issues like:

- #6899

Reply:
I'll check both of versions 😉